sdk/src/wallet/operations/transaction/high_level/create_account.rs

```diff
@@ -98,7 +99,7 @@ where
             }
         }
 
-        let outputs = [account_output_builder.finish_output()?];
+        let outputs = [account_output_builder.finish_output().map_err(BlockError::from)?];
```

Comment:
Can we not add some from impls for the most important sub errors to limit the noise?

Reply:
sure